src/state/prommPools/hooks.ts

```diff
@@ -211,14 +206,14 @@ export function useUserProMMPositions(): UserPositionResult {
       const token0Amount = CurrencyAmount.fromRawAmount(position.pool.token0, position.amount0.quotient)
       const token1Amount = CurrencyAmount.fromRawAmount(position.pool.token1, position.amount1.quotient)
 
-      const token0Usd = parseFloat(token0Amount.toFixed()) * ethPriceUSD * parseFloat(p.pool.token0.derivedETH)
-      const token1Usd = parseFloat(token1Amount.toFixed()) * ethPriceUSD * parseFloat(p.pool.token1.derivedETH)
+      const token0Usd = parseFloat(token0Amount.toFixed()) * prices[token0.address]
```

Comment:
it can be NaN

Reply:
dont worry bro, if it's NAN or 0, it doesnt appears on UI

Reply:
make sure it don't crash in anywhere, I'm afraid of pass NaN to BigInt, Fraction will make crash